K2Bridge.Tests.UnitTests/KustoDAL/KustoResponseParserTests.cs

```diff
@@ -156,6 +157,25 @@ public void ParseElasticResponse_WithAggs_ReturnsElasticResponseWithAggs()
             Assert.AreEqual(2, elasticResult.Aggregations.Collection.Buckets.Count());
         }
 
+        [Test]
```

Comment:
In the visitor file, below, I added a comment about what happens with overlapping ranges
1-100
10-30

Depends on what we decide should be the behavior maybe add a test to validate it

Reply:
The new query should return the same results as the previous one, so there should be no changes to the response parser.